immuni_exposure_ingestion/helpers/risk_level.py

```diff
@@ -64,3 +63,17 @@ def extract_keys_with_risk_level_from_upload(upload: Upload) -> Iterable[Tempora
     )
 
     return keys_at_risk_filtered
+
+
+def set_highest_risk_level_from_upload(
+    keys: Iterable[TemporaryExposureKey],
+) -> Iterable[TemporaryExposureKey]:
+    """
+    :param keys: the keys from the upload.
+    :return: the list of the given keys with risk of transmission set to highest.
+    """
+
+    for key in keys:
+        key.transmission_risk_level = TransmissionRiskLevel.highest
+
+    return keys
```

Comment:
It's an object, so you shouldn't need to return it, as their value is being changed already.

Reply:
AFAIK, you should be able to avoid returning them back, and just use them
```python
    keys_at_risk = [key for key in upload.keys if key.created_at.date() >= first_risky_time]
    set_highest_risk_level_from_upload(keys_at_risk)
    ...  # use keys_at_risk, which will have the transmission_risk_level properly set. 
```